webview/src/experiments/components/table/TableFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { AddAndRemoveBranches } from './commitsAndBranches/AddAndRemoveBranches'
+import styles from './styles.module.scss'
+
+export const TableFooter: React.FC = () => (
+  <tfoot className={styles.footer}>
+    <tr>
+      <td colSpan={99999}>
+        <AddAndRemoveBranches />
```

Comment:
Out of scope, but I'll try to find some time to look into a more button-like design that could possibly look better for actions like these in our table and setup page :)